File: DEV/src/AMSuite2.Web/src/app/core/profile/profile.model.ts
import { build } from '@caiu/core';
import { lookupKeySelector } from '@caiu/http';
import { Store } from '@caiu/store';

import { Accounts } from '../accounts/accounts.model';
import { GroupMember } from '../groups/groups.model';
import { Account, AccountMember } from '../../account/account.model';
import { CurrentUser } from '../../shared/models';
import { Tabs, Tab } from '../../shared/tabs/tabs.model';

export class Profile {
    userId = 0;
    agendaDateRangeId = 10;
    autoSaveEnabled = false;
    emailAddress = '';
    firstName = '';
    generalInfo = '';
    groupId = 0;
    lastName = '';
    userTitle = '';
    warnOnDirty = true;
    newPassword = '';
    confirmPassword = '';
}

export class AccountRequest {
    accountId = 0;
    accountName = '';
    notificationStatus = 'Action Required';
    notificationStatusId = 1;
    notificationType = 'Account Request';
    notificationTypeId = 0;
    reason = '';
    userId = 0;
    userName = '';
}

File: TEST/src/AMSuite2.API/Features/Core/Profile/ProfileModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using AMSuite2.API.Infrastructure.Models;

namespace AMSuite2.API.Features.Core.Profile
{
    public class ProfileModel : BaseEntity
    {
        public CurrentUserModel User { get; set; }
        public int UserId { get; set; }
        public bool AutoSaveEnabled { get; set; }
        public string EmailAddress { get; set; }
        public string FirstName { get; set; }
        public string GeneralInfo { get; set; }
        public string LastName { get; set; }
        public string UserTitle { get; set; }
        public bool WarnOnDirty { get; set; }
        public string NewPassword { get; set; }
        public string ConfirmPassword { get; set; }
    }
}
